docs/docs/agents/mcp.md

```diff
@@ -7,7 +7,7 @@ hide:
   - tags
 ---
 
-# MCP Integration
+# Use MCP
 
 [Model Context Protocol (MCP)](https://modelcontextprotocol.io/introduction) is an open protocol that standardizes how applications provide tools and context to language models. LangGraph agents can use tools defined on MCP servers through the `langchain-mcp-adapters` library.
```

Comment:
I think this first section should be moved into a conceptual doc on the Get started tab and combined with the overview content on the Server API page